### PR TITLE
removing myself - also should add Danish person(s)...

### PR DESCRIPTION
**Pull request by danielskatz**

### CODE REVIEW

File: council.md [hunk 1]
```diff
@@ -49,7 +49,9 @@ The Council has a [*Code of Conduct*](./council/code-of-conduct.md), and you are
 | RSE-AUNZ    | Rowland Mosbergen | rowlandm@practicaldiversity.org |
 | Nordic-RSE  | Anne Claire Fouilloux | annefou@geo.uio.no |
 | Nordic-RSE  | Radovan Bast | radovan.bast@uit.no |
-| US-RSE  | Daniel S. Katz | d.katz@ieee.org |
+<!--
```
Comment (danielskatz) [inline council.md:52]:
```suggestion
<!--
```

File: council.md [hunk 2]
```diff
@@ -49,7 +49,9 @@ The Council has a [*Code of Conduct*](./council/code-of-conduct.md), and you are
...
+| US-RSE  | TBD | TBD |
+-->
```
Comment (danielskatz) [inline council.md:54]:
```suggestion
-->
```

Comment (danielskatz) [inline council.md:54]:
```suggestion
| US-RSE  | Jeffrey Carver | carver@cs.ua.edu
```

File: council.md (hunk 1 above)
Comment (danielskatz) [inline council.md:52]:
```suggestion
```

File: council.md [hunk 3]
```diff
@@ -49,7 +49,8 @@ The Council has a [*Code of Conduct*](./council/code-of-conduct.md), and you are
 | RSE-AUNZ    | Rowland Mosbergen | rowlandm@practicaldiversity.org |
 | Nordic-RSE  | Anne Claire Fouilloux | annefou@geo.uio.no |
 | Nordic-RSE  | Radovan Bast | radovan.bast@uit.no |
-| US-RSE  | Daniel S. Katz | d.katz@ieee.org |
+| US-RSE  | TBD | TBD |
```
Comment (danielskatz) [inline council.md:52]:
```suggestion
```

File: council.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ The Council has a [*Code of Conduct*](./council/code-of-conduct.md), and you are
 | RSE-AUNZ    | Rowland Mosbergen | rowlandm@practicaldiversity.org |
 | Nordic-RSE  | Anne Claire Fouilloux | annefou@geo.uio.no |
 | Nordic-RSE  | Radovan Bast | radovan.bast@uit.no |
-| US-RSE  | Daniel S. Katz | d.katz@ieee.org |
+| US-RSE  | Jeffrey Carver | carver@cs.ua.edu
```
Comment (danielskatz) [inline council.md:52]:
```suggestion
| US-RSE  | Jeffrey Carver | carver@cs.ua.edu |
```

**Review by sdruskat — APPROVED** [review]
Thanks @danielskatz!

**Pull request merged**